README.md

```diff
@@ -1,35 +1,50 @@
-# pyintelowl
+# PyIntelOwl
 
 [![PyPI version](https://badge.fury.io/py/pyintelowl.svg)](https://badge.fury.io/py/pyintelowl)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/intelowlproject/pyintelowl.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/mlodic/pyintelowl/context:python)
 [![CodeFactor](https://www.codefactor.io/repository/github/intelowlproject/pyintelowl/badge)](https://www.codefactor.io/repository/github/intelowlproject/pyintelowl)
 
-Simple Client for the [Intel Owl Project](https://github.com/intelowlproject/IntelOwl)
-
-2 ways to use it:
-* as a library
-* as a command line script
+Robust Python **SDK** and **Command Line Client** for interacting with [IntelOwl](https://github.com/intelowlproject/IntelOwl)'s API.
 
 You can select which analyzers you want to run for every analysis you perform.
 
 For additional help, we suggest to check the ["How to use pyintelowl" Youtube Video](https://www.youtube.com/watch?v=fpd6Kt9EZdI) by [Kostas](https://github.com/tsale).
 
 
-## Generate API key
-You need a valid API key to interact with the IntelOwl server. 
-Keys should be created from the admin interface of [IntelOwl](https://github.com/intelowlproject/intelowl): you have to go in the *Durin* section (click on `Auth tokens`) and generate a key there.
+## Installation
 
-You can use the  with the parameter `-k <api_key>` from CLI
+```bash
+$ pip3 install pyintelowl
+```
 
-#### (old auth method) JWT Token Authentication
-> this auth was available in IntelOwl versions <1.8.0 and pyintelowl versions <2.0.0
+For development/testing, `pip3 install pyintelowl[dev]`
 
-From the admin interface of IntelOwl, you have to go in the *Outstanding tokens* section and generate a token there.
+## Usage
+
+### As Command Line Client
+
+```bash
+$ python3 cli.py -h
+Usage: cli.py [OPTIONS] COMMAND [ARGS]...
+
+Options:
```

Comment:
this is different from what I see:

```
Usage: cli.py [OPTIONS] COMMAND [ARGS]...

Options:
  -d, --debug  Set log level to DEBUG
  -h, --help   Show this message and exit.

Commands:
  analyse              Send new analysis request
  config               Set or view config variables
  get-analyzer-config  Get current state of `analyzer_config.json` from the...
  jobs                 Manage Jobs
  tags                 Manage tags

```
Is it outdated or am I missing something?

Reply:
Yes, changed a few things yesterday that's why it is outdated. There's still much documentation left.